docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -353,7 +353,7 @@ in order for the token to be accepted as valid.
 <2> The `upn` claim is defined by the {mp-jwt} spec as preferred claim to use for the
 `Principal` seen via the container security APIs.
 <3> The `group` claim provides the groups and top-level roles associated with the JWT bearer.
-<4> The `birthday` claim. It can be considered to be a sensitive claim so you may want to consider encrypting the claims, see xref:smallrye-jwt-build.adoc[Generate JWT tokens with SmallRye JWT].
+<4> The `birthday` claim. It can be considered to be a sensitive claim so you may want to consider encrypting the claims, see https://smallrye.io/docs/smallrye-jwt/generate-jwt.html[Generate JWT tokens with SmallRye JWT].
```

Comment:
```suggestion
<4> The `birthday` claim. It can be considered to be a sensitive claim so you may want to consider encrypting the claims, see xref:security-jwt-build.adoc[Generate JWT tokens with SmallRye JWT].
```